build_msvc.bat

```diff
@@ -0,0 +1 @@
+cl.exe /Ox /openmp /I. run.c mman-win32.c
```

Comment:
Sorry noob question but how is this file related to Makefile? What is this file for / is it needed?

Reply:
The Makefile is intended for cross compiling from linux, it will output a .exe. This .bat file would be what you run on windows if you have visual studio installed to generate an .exe from command line compiler cl.exe